Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -313,7 +313,7 @@ Page* WebChromeClient::createWindow(LocalFrame& frame, const WindowFeatures& win
 #endif
     };
 
-    WebFrame* webFrame = WebFrame::fromCoreFrame(frame);
+    RefPtr webFrame = WebFrame::fromCoreFrame(frame);
```

Comment:
Should `WebFrame::fromCoreFrame` just return a `RefPtr<WebFrame>`? It seems like many cases want this.